schema/organization/Person.md

```diff
@@ -0,0 +1,130 @@
+url: https://link.aps.org/doi/10.1103/RevModPhys.3.427
+authors:
+  - Marie Curie <marie@mariecurie.org.uk> (https://www.mariecurie.org.uk/)
+  - Debierne, A.
+  - Eve, A. S.
+  - Geiger, H.
+  - Hahn, O.
+  - Lind, S. C.
+  - Meyer, St.
+  - Rutherford, Ernest (https://en.wikipedia.org/wiki/Ernest_Rutherford)
+  - Schweidler, E.
+```
+
+## Related
+
+### JATS
+
+`Person` is analogous, and structurally similar to, the JATS [`<contrib>`](https://jats.nlm.nih.gov/archiving/tag-library/1.1/element/contrib.html) element.
+JATS includes `<custom-meta>` element, this _"element can be used as an escape-hatch to record additional metadata that a publisher or archive wishes to preserve (so that the intellectual work to capture that content will not be lost) even though there is no direct expression for the metadata in the Tag Set"._
+
+### OpenDocument
+
+`Person` is analogous to the OpenDocument `Author Fields` element which is the elements in a single OpenDocument XML document:
+
+> The [`Author Fields`](http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#__RefHeading__1415310_253892949) elements are:
+>
+> - The <text:author-name> element represents the full name of the author of a document.
+> - The <text:author-initials> element represents the initials of the author of a document.
+
+### HTML5
+
+In HTML5 metadata about the author can be included in two ways.
+
+1. If there is a link to the detailed information about the author (eg. their homepage), `rel="author"` with `link` or `a` [should be used](https://html.spec.whatwg.org/multipage/links.html#link-type-author).
+   Example:
+
+```
+<a href="http://johnsplace.com" rel="author">John</a>
+```
+
+2. If there is no link to the information about the author, `class="author"` attribute should be used with `area`, `span` and so on.
+   Example:
+
+```
+<span class="author">John</span>
+```
+
+### Crossref
+
+`Person` is analogous to the Crossref [`<contributors>`](https://support.crossref.org/hc/en-us/articles/214567746-Authors-and-editors) child element
+[`<person_name>`](https://data.crossref.org/reports/help/schema_doc/4.4.0/relations_xsd.html#http___www.crossref.org_relations.xsd_person_name) which describes a person who contributed to authoring or editing an entity.
+
+### Citation Style Language (CSL)
+
+`Person` is analogous to the CSL JSON Schema for [`author`](https://github.com/citation-style-language/schema/blob/f01ba9c5ec2055e381a38598919a379255c496c5/csl-data.json#L72) items which describe a person who contributed to authoring an entity.
+
+### Summary
+
+The following table summarizes how properties of `Person` relate to other schema.
+
+| `Person`        | Crossref `<person-name>` | CSL `author` | JATS `<contrib>` |
+| --------------- | ------------------------ | ------------ | ---------------- |
+| affiliation     | affiliation              |              | aff              |
+| contactPoints   |                          |              | corresp          |
```

Comment:
JATS `corresp` is actually an attribute which has value "yes" or "no" and so does not correspond to `contactPoint` - which I think we should remove anyway. We should add `address` to this table.

Reply:
`contactPoints` line removed and `address` line added to table.